src/Bridge/Symfony/Validator/EventListener/ValidateListener.php

```diff
@@ -82,6 +83,10 @@ public function onKernelView(ViewEvent $event): void
             $validationGroups = $validationGroups($data);
         }
 
+        if ($this->container instanceof SymfonyContainerInterface && empty($validationGroups)) {
```

Comment:
But this won't work anymore in 3.0 :man_facepalming: :smile: Container will be replaced with a locator.